kramdown_rpf.gemspec

```diff
@@ -31,6 +30,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'kramdown', '~> 1.2', '>= 1.2.0'
   spec.add_dependency 'i18n', '0.8.6'
+  spec.add_dependency 'rouge'
```

Comment:
Don't think we'll need this - we're hoping to remove it from ups in fact.

Reply:
Good point. Will remove.